modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_form.rb

```diff
@@ -194,6 +194,7 @@ def data_must_comply_with_schema
               },
               "phone": {
                 "type": ["string", "null"],
+                "pattern": "^\\\\d{10}$",
```

Comment:
Are there extra backslashes here?

Reply:
so I found out that it needed to be double escaped because it's json in a ruby string.

I think we can deploy this separately from the presentation part.